solidus_importer.gemspec

```diff
@@ -7,17 +7,16 @@ Gem::Specification.new do |s|
   s.name        = 'solidus_importer'
   s.version     = SolidusImporter::VERSION
   s.summary     = 'Solidus Importer'
-  s.description = 'Solidus Importer'
+  s.description = 'Solidus Importer extension to migrate data from other eCommerce systems'
   s.license     = 'BSD-3-Clause'
 
   s.author      = 'Nebulab SRL'
-  s.homepage    = 'https://www.nebulab.it'
-  # s.email       = 'TODO'
+  s.email       = 'contact@solidus.io'
+  s.homepage    = 'https://solidus.io'
```

Comment:
I think the home page should point to the repo's readme, not blocking though, we can deal with that after the merge